Lib/swiginit.swg

```diff
@@ -50,9 +50,12 @@ extern "C" {
 #define SWIGRUNTIME_DEBUG
 #endif
 
+#ifndef SWIG_INIT_CLIENT_DATA_TYPE
+#define SWIG_INIT_CLIENT_DATA_TYPE void *
+#endif
```

Comment:
I can't get my head round why this fallback definition is needed as `SWIG_INIT_CLIENT_DATA_TYPE` looks to be always defined (in javascriptinit.swg). So probably this can be removed. If not I'd like to see it defined to the different types in the same place in javascriptinit.swg so that it is clear when the type is different.

Reply:
It won't be defined in **other** languages, hence that fallback.

(It ought to be late in the day for you too, huh:-)

Reply:
Should I throw in comment mentioned that it can be overridden in language-specific modules?

Reply:
Duh, I missed that it was in a general purpose file and not a v8 specific file. Leave it as is, I'll add a comment. It's otherwise okay for merging, which I'll do.

Reply:
Thanks!

If you feel that I can help with something else related, now or in the future, just tag me. Well, I naturally can't guarantee that I'll have time or that it won't be over my head, but I'll definitely try to help.

Cheers. (Out for today.)

Reply:
One extend this SWIG_INIT_CLIENT_DATA_TYPE approach to recent modifications to Lib/javascript/jsc/javascriptinit.swg. If you don't have time, just say the word.